docs/core/diagnostics/includes/available-counters.md

```diff
@@ -62,12 +62,12 @@ The following counters are published as part of the [ASP.NET Core Kestrel web se
 | Counter | Description |
 |--|--|
 | :::no-loc text="Connection Queue Length"::: (`connection-queue-length`) | The current length of the connection queue |
-| :::no-loc text="Connection Rate"::: (`connections-per-second`) | The number of connections per second to the web server |
+| :::no-loc text="Connection Rate"::: (`connections-per-second`) | The number of connections per update interval to the web server |
```

Comment:
The name `connections-per-second` is now out of sync with the description.

Reply:
Unfortunately, the names of counters are not documentation. They're implementation details. We could change it, but this will cause backward compat issues.